edalize/templates/incisive/Makefile.j2

```diff
@@ -0,0 +1,10 @@
+all: {{ name }}
+
+{{ name }}: {{ name }}.scr
+	ncverilog -C -f {{ name }}.scr +nctop+{{ toplevel }} {% for option in incisive_options %} {{ option }}{% endfor %}
+
+run: {{ name }}
+	ncverilog +nclibdirname+INCA_libs -r worklib.{{ toplevel }}:v {% for plusarg in plusargs %} {{ plusarg }} {% endfor %}{% for option in run_options %} {{ option }}{% endfor %}
```

Comment:
I'm always sceptic towards hard-coded options as they tend to be the backend author's preference in 99% of the cases rather than something really required. Is this INCA_libs thingamob a mandatory option? What does it do?

Reply:
What's `:v` after the toplevel name? Is `worklib` required? Do you need to prefix parameters with the library name as well?

Reply:
My understanding is that this is required in the same way modelsim has that work directory?

Reply:
I think the ``nclibdirname`` is not required, and ``INCA_libs`` is the default anyway. Also, it's usually written like ``-nclibdirname INCA_libs``. I basically never use plusargs with Incisive.